src/lib/airtable/tables/requests.js

```diff
@@ -224,108 +224,3 @@ const removeSlackMeta = (meta) =>
     .omit("slack_channel")
     .thru(JSON.stringify)
     .value();
-
```

Comment:
I like refactoring this out and I think it makes a lot of sense! However, removing `fields` from exports will break a bunch of APIs and UI that are importing these fields from `requests.js`. e.g. https://github.com/sandfort/mutual-aid-app/blob/a090a31523b4d69733cf5bddbbabe27e869dfa08/src/api/delivery-needed/index.js#L10

Can I suggest exporting it to keep this support for now? we can just do `exports.fields = fields`

Reply:
Whoops! I'm actually still exporting `fields`, just doing it from `requestsSchema.js` now instead. I thought I had changed the various imports throughout the app, but I see I missed some...

Reply:
gotcha, i see you updated all the places where its imported, that works too!